v2_bridges/int_on_eos/src/eos/parse_tx_info.rs

```diff
@@ -106,6 +106,7 @@ impl IntOnEosIntTxInfo {
                 let (token, cid): (&str, MetadataChainId) = match eos_address {
                     "wmbt.ptokens" => ("pWOMBAT", MetadataChainId::EthereumMainnet),
                     "btc.ptokens" => ("pBTC", MetadataChainId::BitcoinMainnet),
+                    "eth.ptokens" => ("PETH", MetadataChainId::EthereumMainnet),
```

Comment:
Double checked. It's OK

Reply:
Just noticed right now this is upper case, but not used. Maybe put `pETH` for consistency?